app/util/geo-utils.coffee

```diff
@@ -77,6 +79,24 @@ getLayerForVehicles = ->
     "icon-ignore-placement": true
     "icon-image": "{mode}"
 
+setDistanceToNearestStop = (lat, lon, stops) ->
```

Comment:
I think this should be "getDistanceToNearestStop", so that it has no side effects